sdk/servicebus/azure-messaging-servicebus/src/main/java/com/azure/messaging/servicebus/implementation/ServiceBusAmqpConnection.java

```diff
@@ -11,14 +11,13 @@
 import reactor.core.publisher.Mono;
 
 public interface ServiceBusAmqpConnection extends AmqpConnection {
-
     /**
      * Gets the management node for a Service Bus resource and performs metadata and management operations using it.
      *
      * @param entityPath Name of the Service bus resource.
      * @return A Mono that completes with that resources's management node.
      */
-    Mono<ServiceBusManagementNode> getManagementNode(String entityPath);
+    Mono<ServiceBusManagementNode> getManagementNode(String entityPath, MessagingEntityType entityType);
 
```

Comment:
We do not know always what is the entity type.

Reply:
We do, a management node is tied to an entity path, which is tied to a specific entity.

Reply:
Just want to clarify that user specify "EntityName" and it could be Queue or Topic. So when user provide `EntityName` connectionString, we will not know is it topic or queue. User can send to a Topic/Queue without knowing what is  EntityType.

But you said you are going to infer it, same way  it is done in track1, and that is fine .